requirements.txt

```diff
@@ -1 +1,4 @@
 mkdocs
+
+rdflib~=7.0.0
+SPARQLWrapper~=2.0.0
```

Comment:
This is part of the ODK, you don't need to install :)

Reply:
Bradley wanted to use script itself without running it through make command, so it is kinda needed, isn't it?

Reply:
is there a reason?

Reply:
He can run the command without make but still use ODK, which is recommended.

Reply:
cc @bvarner-ebi